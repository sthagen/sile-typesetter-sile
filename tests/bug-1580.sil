\begin[class=plain,papersize=a7]{document}
\use[module=packages.retrograde,target=v0.15.0]
\neverindent
\nofolios
\use[module=packages.masters]
\use[module=packages.lorem]
\define-master-template[id=mymaster,first-content-frame=myframe]{
	\frame[id=myframe,left=5%pw,right=85%pw,top=5%pw,bottom=100%ph-5%pw]
}
\switch-master[id=mymaster]
\hbox{}
\skip[height=90%fh]
\lorem[words=20]
\end{document}
