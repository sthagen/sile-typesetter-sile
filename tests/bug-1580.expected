Set paper size 	209.7637818	297.6377985
Begin page
Mx 	10.4882
My 	10.4882
My 	272.0958
Set font 	Gentium Plus;10;400;;normal;;;LTR;
T	79 82 85 72 80 w=24.3506	(lorem)
Mx 	39.1334
T	76 83 86 88 80 w=25.2002	(ipsum)
Mx 	68.6282
T	71 82 79 82 85 w=21.9287	(dolor)
Mx 	94.8516
T	86 76 87 w=10.0146	(sit)
Mx 	109.1608
T	68 80 72 87 w=20.6836	(amet)
Mx 	134.1391
T	70 82 81 86 72 87 72 87 88 85 w=44.1602	(consetetur)
Mx 	10.4882
My 	284.7082
T	86 68 71 76 83 86 70 76 81 74 w=42.9980	(sadipscing)
Mx 	55.3948
T	72 79 76 87 85 w=17.4414	(elitr)
Mx 	74.7447
T	86 72 71 w=13.6816	(sed)
Mx 	90.3349
T	71 76 68 80 w=20.5322	(diam)
Mx 	112.7756
T	81 82 81 88 80 92 w=34.3262	(nonumy)
Mx 	149.0103
T	72 76 85 80 82 71 w=29.5508	(eirmod)
New page
Mx 	10.4882
My 	18.0565
T	87 72 80 83 82 85 w=30.3809	(tempor)
Mx 	43.6650
T	76 81 89 76 71 88 81 87 w=35.3271	(invidunt)
Mx 	81.7882
T	88 87 w=8.7402	(ut)
Mx 	93.3244
T	79 68 69 82 85 72 w=25.9863	(labore)
Mx 	122.1067
T	72 87 w=8.0615	(et)
Mx 	132.9642
T	71 82 79 82 85 72 w=26.5479	(dolore)
Mx 	162.3080
T	80 68 w=12.6221	(ma)
Mx 	174.9301
T	16 w=3.3691	(-)
Mx 	10.4882
My 	30.0565
T	74 81 68 w=14.9854	(gna)
Mx 	28.1285
T	68 79 76 84 88 92 68 80 w=38.0127	(aliquyam)
End page
Finish
